code/modules/hydroponics/grown/grass_carpet.dm

```diff
@@ -131,3 +131,18 @@
 	filling_color = "#245c39"
 	bitesize_mod = 3
 	can_distill = FALSE
+	var/value = 0
+
+	//clover
+	/obj/item/reagent_containers/food/snacks/grown/grass/shamrock/Initialize(mapload, obj/item/seeds/new_seed)
+		. = ..()
+		if(prob(0.001)) // 0.001% chance to be a clover
+			name = "four leafed clover"
+			icon_state = "clover"
+			desc = "A rare sought after trinket said to grant luck to it's holder."
+			icon_state = "clover"
```

Comment:
Duplicated line. You might want to remove 141 line one as name-desc order is sane